packages/babel-preset-expo/src/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -290,10 +290,10 @@ console.log("Hey I'm running on the UI thread");
 
 exports[`metro+hermes supports reanimated worklets 1`] = `
 "var _worklet_14307677064221_init_data={code:"function someWorklet_workletJs1(greeting){console.log(\\"Hey I'm running on the UI thread\\");}",location:"[mock]/worklet.js",sourceMap:"{\\"version\\":3,\\"names\\":[\\"someWorklet_workletJs1\\",\\"greeting\\",\\"console\\",\\"log\\"],\\"sources\\":[\\"[mock]/worklet.js\\"],\\"mappings\\":\\"AAAA,SAAAA,uBAAAC,QAAA,EAAAC,OAAA,CAAAC,GAAA,qCACA\\",\\"ignoreList\\":[]}",version:"[GLOBAL]"};var someWorklet=
-function(){var _e=[new global.Error(),1,-27];var someWorklet_workletJs1=function(greeting){
+function(){var _e=[new global.Error(),1,-27];var someWorklet=function(greeting){
```

Comment:
this worklet changes are coming from reanimated bump. it breaks on main as well: https://github.com/expo/expo/actions/runs/11163465127/job/31030481206